commands/firstline/index.js

```diff
@@ -92,7 +103,7 @@ module.exports = {
 			partialReplies: [
 				{
 					bancheck: false,
-					message: `${prefix} first message in this channel was (${sb.Utils.timeDelta(metaData.First_Message_Posted)}):`
+					message: `${prefix} first message in the specified channel was (${sb.Utils.timeDelta(metaData.First_Message_Posted)}):`
 				},
 				{
 					bancheck: true,
```

Comment:
Regarding `Dynamic_Description` - if you'd want to, you can document the command along with the new functionality - check other commands for inspiration

Reply:
I added the dynamic description. 

Let me know what you think of it, if it should be written differently, if the examples are not fitting, and so on 👍